README.md

```diff
@@ -1 +1,39 @@
 # apostrophe-open-graph
+
+Facebook's Open Graph for [ApostropheCMS](http://apostrophecms.org/).
+
+## Installation
+
+```bash
+npm install apostrophe-open-graph --save
+```
+
+## Use
+
+Configure `apostrophe-open-graph` in `app.js`.
+
+```js
+const apos = require('apostrophe')({
+  shortName: 'project',
+  modules: {
+    'apostrophe-open-graph': {}
+  }
+});
+```
+
+If you choose to disable fields for a piece or page you can do so by setting `openGraph: false` on the module. `apostrophe-files`, `apostrophe-global`, `apostrophe-groups`, `apostrophe-images`, `apostrophe-users` have `openGraph: false` configured by default.
+
+```js
+module.exports = {
+  name: 'person',
+  label: 'Person',
+  pluralLabel: 'People',
+  openGraph: false
+};
+```
+
+Add the following to `layout.html` that all of your pages extend, or to `outerLayout.html` if you have one in `apostrophe-templates/views/`.
+
+```nunjucks
+{% include "apostrophe-open-graph:view.html" %}
```

Comment:
This is good, but it requires context. At this point the user can't see that this is going to output meta tags and doesn't know to put it in the head. So give a more complete example, overriding the appropriate block, making the reasonable assumption that they have our outerLayoutBase at the bottom of the stack of turtles.

Reply:
Thanks Tom, good feedback. See here 7e1b021